core/src/main/scala/kafka/common/InterBrokerSendThread.scala

```diff
@@ -118,9 +119,9 @@ abstract class InterBrokerSendThread(name: String,
 
   private def failExpiredRequests(now: Long): Unit = {
     // clear all expired unsent requests
-    val expiredRequests = unsentRequests.removeExpiredRequests(now, unsentExpiryMs)
+    val expiredRequests = unsentRequests.removeExpiredRequests(now, requestTimeoutMs)
```

Comment:
nit: update `def removeExpiredRequests(now: Long, unsentExpiryMs: Long)` to `removeTimedoutRequest(.., requestTimouetMs: Long)`?